### DISCUSSION TITLE
Windows GCC 不能切换成debug模式

### DISCUSSION THREAD

**Discussion by SilhouettesForYou** [xmake-io/xmake#5246] [Q&A] [answered]
<img width="385" alt="image" src="https://github.com/xmake-io/xmake/assets/53727497/dab6993b-a0c6-4185-b15d-2239a3ef59ce">

![image](https://github.com/xmake-io/xmake/assets/53727497/bb4f96ed-df4f-4b5e-be2c-97b28dfce479)

![image](https://github.com/xmake-io/xmake/assets/53727497/d6779c0b-6979-4050-a373-780409813429)

![image](https://github.com/xmake-io/xmake/assets/53727497/202dfb8c-7f2e-415f-b27a-7757abb6fb1e)

xmake.lua
```lua
set_project("OpenGL Tutorial")
add_rules("mode.debug", "mode.releasedbg")

target("HelloOpenGL")
    set_kind("binary")
    add_files("src/HelloOpenGL.cpp")
    add_includedirs("/dependence/glfw-3.4.bin.WIN64/include")
    add_includedirs("/dependence/glew-2.1.0/include")
    add_linkdirs("/dependence/glfw-3.4.bin.WIN64/lib-static-ucrt")
    add_linkdirs("/dependence/glew-2.1.0/lib/Release/x64")
    add_linkdirs("/dependence/glew-2.1.0/bin/Release/x64")
    add_links("glfw3dll", "glew32")
    add_syslinks("opengl32")
```

**Comment by waruqi** [comment, marked as answer]
xmake f 不支持增量配置，你得写全配置命令，xmake f --toochain=gcc -m debug -c
